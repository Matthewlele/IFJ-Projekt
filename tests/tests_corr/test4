# spravne
def _foo (a , b)
  x = (a * b)
  print x
end

def _bar (
  a = 5
  b = 6
  print "Soucin 5 a 6 je "
  _foo a, b
end

_bar
