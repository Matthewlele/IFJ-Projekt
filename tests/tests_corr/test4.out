expected SEMANTIC ERR: line 6
